CMakeLists.txt

```diff
@@ -28,6 +28,8 @@ include(${CMAKE_BINARY_DIR}/conan.cmake)
 
 conan_cmake_configure(REQUIRES openblas/0.3.20
```

Comment:
Why we have this duplication with setup.py. We can get dependencies from setup.py in cmake.